db/external_sst_file_ingestion_job.cc

```diff
@@ -793,6 +793,18 @@ Status ExternalSstFileIngestionJob::GetIngestedFileInfo(
     file_to_ingest->smallest_internal_key.SetFrom(key);
 
     iter->SeekToLast();
+    if (!iter->Valid()) {
```

Comment:
It might be better to check if the CF uses plain table directly? You should be able to check `cfd_->ioptions()->table_factory`

Reply:
Thank you for your suggestion. Maybe the common sense is checking the validity directly. 
I think your point is to check it in a smaller scope?

Reply:
Valid() only tells us that iterator reaches the end, you will need to check iter->status() to see if NotSupported() is returned too. I think checking for plain table gives a better signal here.

Reply:
Got it. I will fix it.